Automattic/sensei
# Add home task list

Part of #5633
 
### Changes proposed in this Pull Request

* This PR adds the presentational components for the home tasks.
* It still doesn't have the transitions to complete the tasks, and to go from the tasks list to the ready state (for the same reason, I didn't add the ready backgrounds yet).

### Testing instructions


* Navigate to the Sensei Home, and make sure the tasks are rendered with the correct styles in all the resolutions.
* Play with the file to simulate different scenarios: `assets/home/tasks/index.js`:
  * Comment all the props passed to `FirstCourse`.
  * Uncomment the Ready component, and comment the others inside the `Section`.


### Screenshot / Video

<img width="1315" alt="Screen Shot 2022-10-10 at 18 58 46" src="https://user-images.githubusercontent.com/876340/194959343-c1f41497-1762-4480-bf6d-e78f7e753cb1.png">

<img width="500" alt="Screen Shot 2022-10-10 at 18 59 09" src="https://user-images.githubusercontent.com/876340/194959354-f4aeb73e-0b48-4ee7-904e-20e3023515c7.png">

<img width="1306" alt="Screen Shot 2022-10-10 at 19 00 18" src="https://user-images.githubusercontent.com/876340/194959364-fef84110-bab6-4bf4-9290-b538e125561c.png">

<img width="279" alt="Screen Shot 2022-10-10 at 19 00 35" src="https://user-images.githubusercontent.com/876340/194959369-8ac1463a-e06a-46fe-a1b1-9f7f1fbdd896.png">

<img width="1328" alt="Screen Shot 2022-10-10 at 19 01 04" src="https://user-images.githubusercontent.com/876340/194959380-42a9606e-dc07-41af-876c-4c446bc8122a.png">
